src/Tribe/Importer/File_Importer.php

```diff
@@ -17,6 +17,7 @@
 	private $updated = 0;
 	private $created = 0;
 	private $skipped = array();
+	private $encoding = array();
```

Comment:
Ahh, right - I see `get_encoding_changes_row_numbers()` now - that makes sense :white_check_mark: